extension/flat_tensor/serialize/serialize.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <executorch/runtime/core/exec_aten/exec_aten.h>
+
+#include <map>
+#include <string>
+
+namespace executorch {
+namespace extension {
+namespace flat_tensor {
+
+/**
+ *
+ * Schema version of the .ptd format. Should be kept in sync with Python.
+ * https://github.com/pytorch/executorch/blob/main/extension/flat_tensor/serialize/serialize.py#L269
```

Comment:
The problem with pointing to a specific line is that it won't be the right line after serialize.py changes over time. I'd just point to the file in general. And since it's in the same dir you could skip the full URL

```suggestion
 * Schema version of the .ptd format. Should be kept in sync with the same value in serialize.py.
```